src/commands/kafkaClusters.test.ts

```diff
@@ -1,31 +1,172 @@
 import * as assert from "assert";
+import * as sinon from "sinon";
 import * as vscode from "vscode";
-import { TEST_CCLOUD_KAFKA_CLUSTER } from "../../tests/unit/testResources/kafkaCluster";
+import { eventEmitterStubs, StubbedEventEmitters } from "../../tests/stubs/emitters";
+import {
+  TEST_CCLOUD_KAFKA_CLUSTER,
+  TEST_CCLOUD_KAFKA_CLUSTER_WITH_POOL,
+} from "../../tests/unit/testResources/kafkaCluster";
 import { CCloudKafkaCluster } from "../models/kafkaCluster";
-import { copyBootstrapServers } from "./kafkaClusters";
+import * as kafkaClusterQuickpicks from "../quickpicks/kafkaClusters";
+import {
+  copyBootstrapServers,
+  selectFlinkDatabaseViewKafkaClusterCommand,
+  selectTopicsViewKafkaClusterCommand,
+} from "./kafkaClusters";
 
-describe("copyBootstrapServers", () => {
-  let _originalClipboardContents: string | undefined;
+describe("kafkaClusters.ts", () => {
```

Comment:
Minor rename suggestion so it isn't initially confused with `quickpicks/kafkaClusters.ts`
```suggestion
describe("commands/kafkaClusters.ts", () => {
```